2023.md

```diff
@@ -152,4 +152,7 @@
 * [Optimizing React Performance: Mastering useMemo Hook](https://medium.com/@jayashakthiperera/optimizing-react-performance-mastering-usememo-hook-6c2dbc686084) by [Jayashakthi Perera](https://medium.com/@jayashakthiperera)
 * [Google Gemini is a Hoax?](https://medium.com/age-of-awareness/google-gemini-is-a-hoax-34dffc45d888?source=friends_link&sk=66199d69bfbbca81abd0a8a029c20558) by [Anjana Samindra Perera](https://anjanasamindraperera.medium.com/)
 * [Visual Editor for MFA with Asgardeo](https://anjanasamindraperera.medium.com/visual-editor-for-mfa-with-asgardeo-59cb02c66fb8?source=friends_link&sk=bc46c40c8c835aaa22ab34c3a9e5f000) by [Anjana Samindra Perera](https://anjanasamindraperera.medium.com/)
-* [Utilizing Adaptive Scripting to Retrieve Specific SAML Assertion Attributes in WSO2 IS Federated Authentication] (https://medium.com/@leonardor_2734/utilizing-adaptive-scripting-to-retrieve-specific-saml-assertion-attributes-in-wso2-is-federated-1bb0508ea5eb) by [Leonardo Roberto Lopez](https://medium.com/@leonardor_2734)
+* [Utilizing Adaptive Scripting to Retrieve Specific SAML Assertion Attributes in WSO2 IS Federated Authentication](https://medium.com/@leonardor_2734/utilizing-adaptive-scripting-to-retrieve-specific-saml-assertion-attributes-in-wso2-is-federated-1bb0508ea5eb) by [Leonardo Roberto Lopez](https://medium.com/@leonardor_2734)
```

Comment:
Let's update the blog count for December as well.

Reply:
Updated with 545b07a8e8cbdd37e44f315612b7b2f5f66bc7f0